src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -422,6 +435,35 @@ export function fetchRequestDetails(id) {
   };
 }
 
+export function fetchConfirmedAppointmentDetails(id) {
+  return async (dispatch, getState) => {
+    const state = getState();
+    const { appointmentDetails } = state.appointments;
+    const futureAppointments = selectFutureAppointments(state);
+    const appointment =
+      appointmentDetails[id] ||
+      futureAppointments?.find(appt => appt.id === id);
+
+    dispatch({
+      type: FETCH_REQUEST_DETAILS,
+    });
+
+    if (appointment) {
+      dispatch({ type: FETCH_REQUEST_DETAILS_SUCCEEDED, appointment, id });
+    } else {
+      // TODO: fetch single appointment
+    }
+
+    // TODO: Why???
```

Comment:
This section isn't necessary for your page, since VA appointments don't have messages.

Reply:
Removed.